doc/source/cluster/kuberay/quickstart.ipynb

```diff
@@ -0,0 +1,616 @@
+   "id": "656a0707",
+   "metadata": {},
+   "source": [
+    "See the [installation instructions](https://docs.ray.io/en/latest/ray-overview/installation.html) for more details. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c0933e2f",
+   "metadata": {},
+   "source": [
+    "### Install kubectl\n",
+    "\n",
+    "We will use kubectl to interact with Kubernetes. Find installation instructions at the [Kubernetes documentation](https://kubernetes.io/docs/tasks/tools/#kubectl).\n",
+    "\n",
+    "### Access a Kubernetes cluster\n",
+    "\n",
+    "We will need access to a Kubernetes cluster. There are two options:\n",
+    "1. Configure access to a remote Kubernetes cluster\n",
+    "**OR**\n",
+    "\n",
+    "2. Run the examples locally by [installing kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation). Start your [kind](https://kind.sigs.k8s.io/) cluster by running the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c764b3ad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! kind create cluster"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "278726e0",
+   "metadata": {},
+   "source": [
+    "To run the example in this guide, make sure your Kubernetes cluster (or local Kind cluster) can accomodate\n",
+    "additional resource requests of 3 CPU and 2Gi memory. \n",
+    "\n",
+    "## Deploying the KubeRay operator\n",
+    "\n",
+    "Deploy the KubeRay Operator by cloning the KubeRay repo and applying the relevant configuration files from the master branch.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "43c66922",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# After the KubeRay 0.3.0 release branch cut, this documentation will be updated to refer to the 0.3.0 branch.\n",
+    "! git clone https://github.com/ray-project/kuberay\n",
+    "\n",
+    "# This creates the KubeRay operator and all of the resources it needs.\n",
+    "! kubectl create -k ray-operator/config/default\n",
+    "\n",
+    "# Note that we must use \"kubectl create\" in the above command. \"kubectl apply\" will not work due to https://github.com/ray-project/kuberay/issues/271"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "522f9a97",
+   "metadata": {},
+   "source": [
+    "Confirm that the operator is running in the namespace `ray-system`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dfec8bba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! kubectl -n ray-system get pod --selector=app.kubernetes.io/component=kuberay-operator\n",
+    "\n",
+    "# NAME                                READY   STATUS    RESTARTS   AGE\n",
+    "# kuberay-operator-557c6c8bcd-t9zkz   1/1     Running   0          XXs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3f0d3d17",
+   "metadata": {},
+   "source": [
+    "### Namespace-scoped operator\n",
```

Comment:
This seems like an advanced use case and should probably not be part of the quickstart? Or have it further down if it is important :)

Reply:
It could also be put into an "advanced" box in sphinx

Reply:
This should be shifted into a different page and linked from here. On the other hand, that's already what's done here.
I think later we can shift this into another place in the Ray docs and spell out the details there.  

It _is_ an issue many people encounter when working in a permissions-limited environment and something that's explicitly spelled out in the docs for the old operator.

Reply:
Keeping it here, for now.